src/api/xxAMIDOxx.xxSTACKSxx.API.ComponentTests/Fixtures/CreateCategoryFixture.cs

```diff
@@ -35,7 +35,7 @@ protected override void RegisterDependencies(IServiceCollection collection)
         {
             base.RegisterDependencies(collection);
 
-            // Mocked external dependencies, the setup should 
+            // Mocked external dependencies, the setup should
```

Comment:
Just a question:

When doing multiline comments would it not be better to use:
`/*
            * <insert here>
            */ `

Reply:
No. Multiline comments are _generally_ fine for copyrights/big chunks of texts at the very top of files, but for comments in actual code they're a relic. The single line comments are always better for readability. Also multiline comments have weird nesting issues and are error prone. Also Microsoft are clear in the convention - https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#commenting-conventions

Reply:
Ah okay. Thanks. I hadn't seen that. I will update myself then :)